Automattic/simplenote-ios
# Remove Trailing Comma on Tag

Fixes [#1120](https://github.com/Automattic/simplenote-ios/issues/1120)
### Fix
When entering tags and separating via `, `, tags added had a trailing comma. Removing that comma to be on par with other platforms.

### Test
1. Go to note.
2. Tap on tag field.
3. Enter `tag, `.
4. See that tag is entered as `tag` instead of `tag,`.
5. Check that tags list has tag as `tag`, not `tag,`.

### Screenshots
![Feb-17-2021 14-37-55](https://user-images.githubusercontent.com/13263478/108265413-5ac18b80-712e-11eb-84c8-7b88ce105c1f.gif)

<details>
<summary>Old screenshots</summary>

#### Trailing comma removed:
![tagbugged](https://user-images.githubusercontent.com/13263478/107430390-b49cd280-6aea-11eb-96f6-6f01b4f6f521.gif)

#### Mid-word comma still present:
![tagbug-mid](https://user-images.githubusercontent.com/13263478/107429918-258fba80-6aea-11eb-8aba-87b0e83dacd0.gif)
</details>

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in 63c887e5 with: 
`Removing trailing comma from tags when used as delimiter #1120`
